src/decoders/default.ts

```diff
@@ -0,0 +1,15 @@
+import { Decoder, SensorStruct } from 'src/helpers/types';
+
+const defaultSensorStruct: SensorStruct = [
+  { key: 'sample_count', dataType: 'uint16_t', display: false },
+  { key: 'mean', dataType: 'float', display: true },
+  { key: 'stdev', dataType: 'float', display: false },
+];
+
+export const defaultDecoder: Decoder = {
```

Comment:
Let's use this file to explicit a bit more what the expected format is for the structure of a decoder